### PR TITLE
Add additional properties to zwave_js device info panel

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
Adding a new property to the device info panel to indicate whether the device is a Z-Wave Plus device or not. Also updated the is secure property to indicate the highest security class instead

Related to https://github.com/home-assistant/core/pull/56927 - that should be merged before this one.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/zwave_js.ts [hunk 1]
```diff
@@ -83,6 +83,9 @@ export interface ZWaveJSNodeStatus {
   node_id: number;
   ready: boolean;
   status: number;
+  is_routing: boolean | null;
+  is_secure: boolean | null;
```
Comment (d1b) [inline src/data/zwave_js.ts:87]: Can we show `is_secure` information on the device page? (the information already seems to be present in server responses but we aren't showing it at this stage from what I see/can tell :-) , perhaps I should open a separate issue for that? )

Comment (bramkragten) [reply src/data/zwave_js.ts:87]: https://github.com/home-assistant/frontend/pull/10135

File: src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts [hunk 2]
```diff
@@ -116,6 +116,14 @@ export class HaDeviceInfoZWaveJS extends LitElement {
...
           : this.hass.localize("ui.common.no")}
       </div>
+      <div>
+        ${this.hass.localize(
+          "ui.panel.config.zwave_js.device_info.zwave_plus"
+        )}:
+        ${this._node.zwave_plus_version
+          ? this.hass.localize("ui.common.yes")
+          : this.hass.localize("ui.common.no")}
+      </div>
```
Comment (kpine) [inline src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts:126]: Would be preferable to show the actual Z-Wave Plus version, somethingg "No", "1" or "2".

File: src/data/zwave_js.ts [hunk 3]
```diff
@@ -177,6 +178,15 @@ export interface RequestedGrant {
 
 export const nodeStatus = ["unknown", "asleep", "awake", "dead", "alive"];
 
+export const securityClass = {
```
Comment (bramkragten) [inline src/data/zwave_js.ts:181]: We already have this here: <[`69b8179` (#10132)](https://github.com/home-assistant/frontend/pull/10132/commits/69b8179ec20ab15a18a595077bf6f42060a10eba#diff-b477815d091e8935bc5d8576f815ee7cccb7373428ea4ffc955a965ce6a0684dR44)>

File: src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts [hunk 4]
```diff
@@ -117,12 +118,20 @@ export class HaDeviceInfoZWaveJS extends LitElement {
...
-          : this._node.is_secure === false
-          ? this.hass.localize("ui.common.no")
-          : this.hass.localize("ui.panel.config.zwave_js.device_info.unknown")}
+        ${this.hass.localize(
+          "ui.panel.config.zwave_js.device_info.highest_security"
+        )}:
+        ${this._node.highest_security_class !== null
+          ? this.hass.localize(
+              `ui.panel.config.zwave_js.highest_security.${
+                securityClass[this._node.highest_security_class]
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts:127]: `SecurityClass[this._node.highest_security_clas]`

File: src/translations/en.json [hunk 5]
```diff
@@ -2787,6 +2787,14 @@
             "dead": "Dead",
             "alive": "Alive"
           },
+          "highest_security": {
+            "temporary": "Temporary",
+            "none": "None",
+            "s2_unauthenticated": "S2 Unauthenticated",
+            "s2_authenticated": "S2 Authenticated",
+            "s2_access_control": "S2 Access Control",
+            "s0_legacy": "S0 (Legacy)"
```
Comment (bramkragten) [inline src/translations/en.json:2796]: These are already at: [`69b8179` (#10132)](https://github.com/home-assistant/frontend/pull/10132/commits/69b8179ec20ab15a18a595077bf6f42060a10eba#diff-e67939fd25c650222db710f18764d10ae69454b0e8ad680f5e10177c2db93ceaR2814-R2829)

File: src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts (hunk 4 above)
Comment (raman325) [reply src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts:127]: well I'll be, I had no idea this is how enums worked

File: src/translations/en.json [hunk 6]
```diff
@@ -2799,24 +2801,27 @@
...
-                "title": "S2 Access Control",
-                "description": "Example: Door Locks and Garage Doors"
-              },
-              "S0_Legacy": {
-                "title": "S0 Legacy",
-                "description": "Example: Legacy Door Locks without S2 support"
-              }
+            "interview_failed": "The device interview failed. Additional information may be available in the logs."
+          },
+          "security_classes": {
```
Comment (bramkragten) [inline src/translations/en.json:2806]: This will need an update to Lokalise after it is merged.

Comment (raman325) [reply src/translations/en.json:2806]: do I need to do anything about that? and do you think I should have kept it where it was instead?

Comment (bramkragten) [reply src/translations/en.json:2806]: No, it is fine :-) And I'll handle it

**Review by bramkragten — APPROVED** [review]

**Comment by kpine** [comment]
Is this OK to merge? It would be a nice addition to the next release.

**Pull request merged**